logger/xp.py

```diff
@@ -224,6 +226,10 @@ def from_json(self, filename):
 
     def to_visdom(self, visdom_opts=None, xlabel=None):
         self.plotter = Plotter(self, visdom_opts, xlabel)
+        # restore visdom options that have been saved (if experiment loaded from file)
+        if hasattr(self, 'visdom_win_opts'):
```

Comment:
Do you want to remove `self.visdom_win_opts` after doing that to avoid reloading it again and again?

Reply:
Fixed in new commit with a `pop` on `self.__dict__`